extensions/3rdparty/LyricWiki/LyricFind/LyricFindTrackingService.class.php

```diff
@@ -147,7 +147,9 @@ public function track($amgId, $gracenoteId, Title $title) {
 				case self::CODE_LYRIC_IS_INSTRUMENTAL:
 				case self::CODE_LYRIC_IS_AVAILABLE:
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce this "switch/case" number of lines from 11 to at most 10, for example by extracting code into function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1151)